### ISSUE TITLE
复现问题

### ISSUE THREAD

**Issue by MOCUISHLE-AC** [GrokCV/SeRankDet#2]
您好，十分感谢您在红外小目标领域的贡献。我十分想follow您的工作，但是遇到了一些复现的问题。
~~~
data = dict(
    dataset_type='IRSTD1k',
    data_root=
    '/data_set/SeRankDet_format/IRSTD-1k/',
    base_size=512,
    crop_size=512,
    data_aug=True,
    suffix='png',
    num_workers=8,
    train_batch=4,
    test_batch=4,
    train_dir='trainval',
    test_dir='test')
log_config = dict(
    interval=50,
    hooks=[
        dict(type='TextLoggerHook', by_epoch=True),
        dict(type='TensorboardLoggerHook')
    ])
dist_params = dict(backend='nccl')
log_level = 'INFO'
load_from = None
resume_from = None
workflow = [('train', 1)]
cudnn_benchmark = True
find_unused_parameters = True
random_seed = 42
gpus = 1
optimizer = dict(
    type='AdamW',
    setting=dict(lr=0.0001, weight_decay=0.01, betas=(0.9, 0.999)))
optimizer_config = dict()
lr_config = dict(
    policy='PolyLR',
    warmup='linear',
    power=0.9,
    min_lr=0.0001,
    warmup_epochs=5)
runner = dict(type='EpochBasedRunner', max_epochs=500)
checkpoint_config = dict(by_epoch=False, interval=1)
evaluation = dict(epochval=1, metric='mIoU', pre_eval=True)
model = dict(
    name='UNet',
    type='EncoderDecoder',
    pretrained=None,
    backbone=dict(type=None),
    decode_head=dict(type='SeRankDet', deep_supervision=True),
    loss=dict(type='SoftIoULoss'))
develop = dict(
    source_file_root=
    '/SeRankDet/model/SeRankDet/SeRankDet.py')
~~~
我已经尽量保持config与您提供的log一致，但是在IRSTD-1k上，IoU只有0.7010，请问这个是合理的波动吗？
PS：想问问您是否尝试过多卡训练，我使用2张3090，为了保持和单卡config 一致，我将train_batch= 2，进行了实验，但是IRSTD-1k上，IoU只有0.6825，config如下：
~~~
data = dict(
    dataset_type='IRSTD1k',
    data_root=
    'SeRankDet_format/IRSTD-1k/',
    base_size=512,
    crop_size=512,
    data_aug=True,
    suffix='png',
    num_workers=8,
    train_batch=2,
    test_batch=4,
    train_dir='trainval',
    test_dir='test')
log_config = dict(
    interval=10,
    hooks=[
        dict(type='TextLoggerHook', by_epoch=True),
        dict(type='TensorboardLoggerHook')
    ])
dist_params = dict(backend='nccl')
log_level = 'INFO'
load_from = None
resume_from = None
workflow = [('train', 1)]
cudnn_benchmark = True
find_unused_parameters = True
random_seed = 42
gpus = 2
optimizer = dict(
    type='AdamW',
    setting=dict(lr=0.0001, weight_decay=0.01, betas=(0.9, 0.999)))
optimizer_config = dict()
lr_config = dict(
    policy='PolyLR',
    warmup='linear',
    power=0.9,
    min_lr=0.0001,
    warmup_epochs=5)
runner = dict(type='EpochBasedRunner', max_epochs=500)
checkpoint_config = dict(by_epoch=False, interval=1)
evaluation = dict(epochval=1, metric='mIoU', pre_eval=True)
model = dict(
    name='UNet',
    type='EncoderDecoder',
    pretrained=None,
    backbone=dict(type=None),
    decode_head=dict(type='SeRankDet', deep_supervision=True),
    loss=dict(type='SoftIoULoss'))
develop = dict(
    source_file_root=
    'SeRankDet/model/SeRankDet/SeRankDet.py')
~~~
最后，再次感谢您提供的codebase，十分期待您的回答

**Comment by YimianDai** [comment]
非常感谢您的反馈！

每个 Best Performance 的 Checkpoint 我们之前已经上传了，接下来我们会把每个数据集上 Best Performance 的 Inference 命令更新一下，方便您复现（目前我们已经给出了 IRSTD1K 上的示例）。

多卡这个没有尝试过，我们只在单卡上训练。
